@xen-orchestra/lite/src/components/Select.tsx

```diff
@@ -44,10 +46,19 @@ const Select = withState<State, Props, Effects, Computed, ParentState, ParentEff
         options?.map(item => {
           const label =
             optionRenderer === undefined
-              ? item.name ?? item.label ?? item.name_label
+              ? item.name ?? item.label ?? item.name_label ?? (typeof item === 'string' || typeof item === 'number')
+                ? item
+                : 'Unknown'
```

Comment:
https://github.com/vatesfr/xen-orchestra/pull/5863#discussion_r785866944
I don't think this would be a good behavior, we're not supposed to have an object here and if it is an object, we should probably throw an error or make it clear to the **developer** that it's not normal.